src/main/webapp/jsp/instructorFeedbackQuestionSubmissionEdit.jsp

```diff
@@ -1,90 +1,36 @@
-
-                <jsp:include page="<%= Const.ViewURIs.FEEDBACK_SUBMISSION_EDIT %>" />
-
-                <div class="bold align-center">
-                    <%
-                        if (data.bundle.questionResponseBundle.isEmpty()) {
-                    %>
-                            There are no questions for you to answer here!
-                    <%
-                        } else if (data.isPreview() || !data.isSessionOpenForSubmission()) {
-                    %>
-                            <input disabled="disabled" type="submit" class="btn btn-primary center-block" id="response_submit_button" data-toggle="tooltip" data-placement="top" title="<%= Const.Tooltips.FEEDBACK_SESSION_EDIT_SAVE %>" value="Submit Feedback">
-                    <%
-                        } else {
-                    %>
-                            <input type="submit" class="btn btn-primary center-block" id="response_submit_button" data-toggle="tooltip" data-placement="top" title="<%= Const.Tooltips.FEEDBACK_SESSION_EDIT_SAVE %>" value="Submit Feedback">
-                    <%
-                        }
-                    %>
-                </div>
-                <br><br>
-            </form>
-        </div>
-
-        <div id="frameBottom">
-         
-        </div>
-    </body>
-</html>
+<%@ taglib uri="http://java.sun.com/jsp/jstl/core" prefix="c" %>
+<%@ taglib tagdir="/WEB-INF/tags" prefix="t" %>
+<%@ taglib tagdir="/WEB-INF/tags/instructor" prefix="ti" %>
+<%@ taglib tagdir="/WEB-INF/tags/instructor/feedbackSubmissionEdit" prefix="instructorFeedbackSubmissionEdit" %>
+
+<c:set var="jsIncludes">
+    <script type="text/javascript" src="/js/feedbackSubmissionsEdit.js"></script>
+    <script type="text/javascript" src="/js/student.js"></script>
+</c:set>
+
+<c:if test="${data.headerHidden}">
+    <c:set var="altHeader">
+        <c:choose>
+            <c:when test="${data.preview}">
+			    <nav class="navbar navbar-default navbar-fixed-top">
+                    <h3 class="text-center">Previewing Session as Instructor ${data.previewInstructor.name} (${data.previewInstructor.email})</h3>
+			    </nav>
+            </c:when>
+            <c:otherwise>
+                <%-- For when header is hidden but it is not a preview --%>
+                <nav class="navbar navbar-default navbar-fixed-top" style="display: none;">
+                </nav>
+            </c:otherwise>
+        </c:choose>
+    </c:set>
+</c:if>
+
+<c:set var="altFooter">
+    <%-- Cannot be empty to prevent default footer --%>
+    <span style="display:none"></span>
+</c:set>
+
+<ti:instructorPageCustom pageTitle="TEAMMATES - Submit Feedback Question" bodyTitle="Submit Feedback Question" jsIncludes="${jsIncludes}" altNavBar="${altHeader}" altFooter="${altFooter}">
+    <instructorFeedbackSubmissionEdit:feedbackQuestionSubmissionForm feedbackSubmissionForm="${data}"/>
```

Comment:
If there's only going to be a single line, it may be better to just not put it into a tag file.
You'll then need to store constants as attributes, though.